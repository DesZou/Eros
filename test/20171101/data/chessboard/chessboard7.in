7
o*ooooo
*ooooo*
oooooo*
ooooooo
*oooooo
ooo*ooo
ooooo*o
