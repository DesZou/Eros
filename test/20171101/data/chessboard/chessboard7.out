1
35940377
276359538
541504569
192675840
0
0
0
